style.css

```diff
@@ -0,0 +1,1014 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600&family=Poppins:wght@100;200;300;400;500;600;700;800;900&family=Raleway:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
```

Comment:
If you specify multiple font weights at the end of the URL, and if you try and use the font and don't set a custom weight on the element, then the browser simply will set the font weight mentioned at the end of the url to all. Also including extra weights will increase font load time, so you should only include what you intend on using. 

- [ ] [OPTIONAL]  As we can see the font weight of all the content on your website is `bold`. It would be better if you could simplify your google font import and adjust the font size to make your design look like [behance design guideline](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)